roles/homebrew/defaults/main.yml

```diff
@@ -33,13 +33,11 @@ homebrew:
     - wget
   casks:
     - diffmerge
-    - firefox
-    - flash-npapi
+    - firefox # Note that it often hangs here and so you need to install manually with `brew cask install xxx`
+    - flash-npapi # Note that it often hangs here and so you need to install manually with `brew cask install xxx`
```

Comment:
For these, see also the note starting with "If running the above code hangs..." at:
https://agworld.atlassian.net/wiki/spaces/DEV/pages/567804018/Laptop+Setup

@pragmatic , if you get some time, it would be great if you could dig into why this regularly hangs.

Reply:
Not sure _why_ and not really able to diagnose at this point, though sometimes a workaround can be to manually `brew cask install` something first, before running the Ansible playbook.

Reply:
Unfortunately that doesn't work either @pragmatic . It failed for firefox for me. Then I installed it with brew cask manually. And then it failed again for flash-npapi.
No idea why, although I didn't turn on verbose logging or anything; perhaps it's something simple. I just don't have the time to look into it right now.